src/traffic_cache_tool/CacheDefs.cc

```diff
@@ -216,7 +218,7 @@ Stripe::Chunk::clear()
 
 Stripe::Stripe(Span *span, Bytes start, CacheStoreBlocks len) : _span(span), _start(start), _len(len)
 {
-  ts::bwprint(hashText, "{} {}:{}", span->_path.path(), _start.count(), _len.count());
+  ts::bwprint(hashText, "{} {}:{}", span->_path.string(), _start.count(), _len.count());
```

Comment:
This is wrong - we need to update either `ts::file` or `BufferWriterFormat` to make this unneeded. I have another PR up that adds the `view` method to `ts::file::path` which should be used to support this.